### PR TITLE
docs(core): add an entry to the glossary about structural directive shorthand

### PR DESCRIPTION
**Pull request by tortuvshin**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #39784

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
Thanks for this update. I think it is very useful.
Can you squash the two commits into one that follows our commit syntax (which the first commit did), please.

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: global-docs-approvers, public-api

File: aio/content/guide/glossary.md [hunk 1]
```diff
@@ -581,6 +581,14 @@ To learn more, see [Lifecycle Hooks](guide/lifecycle-hooks).
 
 {@a M}
 
+{@a microsyntax}
+
+## microsyntax
+
+The Angular microsyntax lets you configure a directive in a compact, friendly string. The microsyntax parser translates that string into attributes on the <ng-template>.
```
Comment (petebacondarwin) [inline aio/content/guide/glossary.md:588]:
```suggestion
The Angular microsyntax lets you configure a directive in a compact, friendly string. The microsyntax parser translates that string into attributes on the `<ng-template>`.

I liked that you marked this ng-template as code. Can you amend your single commit with this additional change?
```

**Review by petebacondarwin — CHANGES_REQUESTED** [review]

**Comment by petebacondarwin** [comment]
Thanks for the updates. What we actually need is either a single commit in this PR that contains all the changes, or two commits where the second is a "fixup" commit of the first. You can read more about fixup commits here: https://github.com/angular/angular/blob/master/docs/FIXUP_COMMITS.md

Please let me know if you need any further help.

**Comment by tortuvshin** [comment]
> Thanks for this update. I think it is very useful.
> Can you squash the two commits into one that follows our commit syntax (which the first commit did), please.
I have merged the two commits

**Review by petebacondarwin — APPROVED** [review]

**Review by AndrewKushnir — APPROVED** [review]
Reviewed-for: public-api

**Review by kapunahelewong — APPROVED** [review]
LGTM! Thank you so much, @tortuvshin!

Reviewed-for: global-docs-approvers

edit: oops dup :)

**Comment by jelbourn** [comment]
Took a look at this, which led me to look at the more in-depth docs about the "microsyntax" concept. Rather than adding this term to the glossary, I would prefer to eliminate this invented term completely and describe the concept as "Structural directive syntax". IMO the "micro" qualifier here is kind of meaningless. 

@AndrewKushnir @petebacondarwin @mgechev  thoughts?

**Comment by petebacondarwin** [comment]
I like the term microsyntax, although I am not strongly wedded to it. It makes a distinction to the "longhand" syntax for structural directives.

**Comment by AndrewKushnir** [comment]
@jelbourn good point!

I believe the "microsyntax" term is quite vague by itself and kind of hints that it's generic and can be used in other places when in fact it works only for structural directives. I like the idea of getting rid of the "microsyntax" in favor of "Structural directive syntax" :)

**Comment by AndrewKushnir** [comment]
@kapunahelewong could you please help with the next steps here? Should the "microsyntax" term be updated in a context of this PR or it should be landed only after #40015? Thank you.

**Comment by kapunahelewong** [comment]
> @kapunahelewong could you please help with next steps? Should the "microsyntax" term be updated in a context of this PR or it should be landed only after #40015? Thank you.

Sure! @tortuvshin, your PR prompted some offline conversation about how best to refer to this concept And we decided on using the term `structural directive shorthand`. Would you be willing to change it from `microsyntax` to `structural directive shorthand` and move it to the "S" section?

**Comment by tortuvshin** [comment]
> > @kapunahelewong could you please help with next steps? Should the "microsyntax" term be updated in a context of this PR or it should be landed only after #40015? Thank you.
> 
> Sure! @tortuvshin, your PR prompted some offline conversation about how best to refer to this concept And we decided on using the term `structural directive shorthand`. Would you be willing to change it from `microsyntax` to `structural directive shorthand` and move it to the "S" section?

Hello. I will now change microsyntax to a structural directive shorthand

**Pull request closed without merging**